### PR TITLE
Dev screpyy96 welcome

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

**Review by marianzburlea — APPROVED** [review]
💯

File: src/_asset/demo-files/demo.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+  font-size: 1em;
+  line-height: 1.5;
+  color: #555;
+  background: #fff;
+}
+h1 {
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:10]: Empty line needed for readability

File: src/_asset/demo-files/demo.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  font-size: 1.5em;
+  font-weight: normal;
+}
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:13]: Empty line needed for readability

File: src/_asset/demo-files/demo.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+small {
+  font-size: .66666667em;
+}
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:16]: Empty line needed for readability

File: src/_asset/demo-files/demo.css [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+a {
+  color: #e74c3c;
+  text-decoration: none;
+}
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:20]: Empty line needed for readability

File: src/_asset/demo-files/demo.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+a:hover, a:focus {
+  box-shadow: 0 1px #e74c3c;
+}
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:23]: Empty line needed for readability

File: src/_asset/demo-files/demo.css [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+.bshadow0, input {
+  box-shadow: inset 0 -2px #e7e7e7;
+}
```
Comment (MariaTeo) [inline src/_asset/demo-files/demo.css:26]: And so on. 😅

File: src/_asset/font/style.scss [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon__arrow-down {
+  &:before {
+    content: $icon__arrow-down; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:33]: Again Empty line needed for readability

File: src/_asset/font/style.scss [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+.icon__arrow-up {
+  &:before {
+    content: $icon__arrow-up; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:38]: Empty line needed for readability

File: src/_asset/font/style.scss [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+.icon__book {
+  &:before {
+    content: $icon__book; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:43]: And so on

**Review by MariaTeo — APPROVED** [review]
Just the empty lines needed. Good job overall. ☺️👌🏼

**Pull request merged**